pyodide_build/pywasmcross.py

```diff
@@ -256,6 +260,34 @@ def handle_command(line, args, dryrun=False):
             arg = arg[:-3] + '.wasm'
             output = arg
 
+        # Fix for scipy to link to the correct BLAS/LAPACK files
+        if arg.startswith('-L') and 'CLAPACK-WA' in arg:
+            out_idx = line.index('-o')
+            out_idx += 1
+            module_name = line[out_idx]
+            module_name = Path(module_name).name.split('.')[0]
+
+            lapack_dir = arg.replace('-L', '')
+            # For convinience we determine needed scipy link libraries
+            # here, instead of in patch files
+            link_libs = ['F2CLIBS/libf2c.bc', 'blas_WA.bc']
+            if module_name in ['_flapack', '_flinalg', '_calc_lwork',
+                               'cython_lapack', '_iterative', '_arpack']:
+                link_libs.append('lapack_WA.bc')
```

Comment:
We only link LAPACK only when it is needed: scipy by default links BLAS and LAPACK if either is needed (e.g. in `_fblas`) as with dynamic linking it's less of an issue.